com.unity.render-pipelines.universal/Editor/Camera/UniversalRenderPipelineCameraStackUI.cs

```diff
@@ -0,0 +1,296 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using NUnit.Framework;
+using UnityEditor.SceneManagement;
+using UnityEditorInternal;
+using UnityEngine;
+using UnityEngine.Rendering.Universal;
+
+namespace UnityEditor.Rendering.Universal
+{
+    internal class UniversalRenderPipelineCameraStackUI
+    {
+        static readonly List<CameraRenderType> s_ValidCameraTypes = new List<CameraRenderType>
```

Comment:
I would suggest re-adding the comment " // This is the valid list of types, so if we need to add more types we just add it here."